pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -141,14 +142,16 @@ stdenv.mkDerivation rec {
       mkdir -p $out/bin/
       ln -s $out/lib/teamspeak/ts3client $out/bin/ts3client
 
+      runHook postInstall
+    '';
+
+  preFixup =
+    ''
       wrapProgram $out/bin/ts3client \
```

Comment:
Are we doing now double wrapping here?

Reply:
I solved that particular problem in a later commit. One step at a time.